internal/controller/postgrescluster/pgbackrest_test.go

```diff
@@ -344,6 +344,7 @@ restartPolicy: Always
 schedulerName: default-scheduler
 securityContext:
   fsGroup: 26
+  fsGroupChangePolicy: OnRootMismatch
```

Comment:
Double-check this -- why here now?

Reply:
:thinking: That's a good question, the change to `OnRootMismatch` should have required this update previously, iiuc. Might be worth discussing with the team.

Reply:
L187 skips when `strings.EqualFold(os.Getenv("USE_EXISTING_CLUSTER"), "true")`, so this has only been running in local envtest. CI for that has been pinned to a specific version for a while.

I think this is a candidate for splitting into "generate" methods.

Reply:
Right, k8s 1.19 was the only version where this wasn't true and the only version being tested in our GH test action